app/middlewares/users/verify_signs_up.js

```diff
@@ -0,0 +1,25 @@
+const { badRequest_Error } = require('../../errors');
+const logger = require('../../logger');
+
+exports.verifyEmail = (req, res, next) => {
```

Comment:
According to the card requirements you should validate more fields to know if they have data, in this case I recommend using the express-validator library that the project has to validate the schema fields